src/main/java/com/quizapp/enterprise/controllers/GameController.java

```diff
@@ -90,7 +95,9 @@ public PlayerJoinEvent send(PlayerJoinEvent message) throws Exception {
         newPlayer.setPlayerUsername(message.getPlayerName());
         newPlayer.setHost(false);
         gameService.joinGame(message.getGameId(), newPlayer);
-        messagingTemplate.convertAndSend("/game1/messages/" + message.getGameId(), message);
+        StringBuilder messagesGameIdString = new StringBuilder("/game1/messages/")
+                .append(message.getGameId());
+        messagingTemplate.convertAndSend(messagesGameIdString.toString(), message);
```

Comment:
refactored the string concatenation to use String builder with append.
This change was recommended to by "5. Use StringBuilder or StringBuffer for String Concatenation " section of the java coding best practices summary referenced below.

source: https://xperti.io/blogs/java-coding-best-practices/